extensions/default/src/Panels/PanelStudyBrowser.tsx

```diff
@@ -286,34 +286,36 @@ function _mapDisplaySets(displaySets, thumbnailImageSrcMap) {
   const thumbnailDisplaySets = [];
   const thumbnailNoImageDisplaySets = [];
 
-  displaySets.forEach(ds => {
-    const imageSrc = thumbnailImageSrcMap[ds.displaySetInstanceUID];
-    const componentType = _getComponentType(ds.Modality);
-
-    const array =
-      componentType === 'thumbnail'
-        ? thumbnailDisplaySets
-        : thumbnailNoImageDisplaySets;
-
-    array.push({
-      displaySetInstanceUID: ds.displaySetInstanceUID,
-      description: ds.SeriesDescription || '',
-      seriesNumber: ds.SeriesNumber,
-      modality: ds.Modality,
-      seriesDate: ds.SeriesDate,
-      seriesTime: ds.SeriesTime,
-      numInstances: ds.numImageFrames,
-      countIcon: ds.countIcon,
-      StudyInstanceUID: ds.StudyInstanceUID,
-      componentType,
-      imageSrc,
-      dragData: {
-        type: 'displayset',
+  displaySets
+    .filter(ds => !ds.excludeFromThumbnailBrowser)
```

Comment:
This is still a worthwhile change, as being able to have hidden display sets is useful